config/swagger.yaml

```diff
@@ -965,6 +965,8 @@ definitions:
         properties:
           data_schema:
             type: string
+          hash:
+            type: string
```

Comment:
When you merge with master later on, please use  $ref: '#/definitions/EncodedHash'